internal/runners/languages/languages.go

```diff
@@ -24,17 +24,16 @@ func NewLanguages(prime primeable) *Languages {
 	}
 }
 
-// Listing represents the output data of a list of languages.
-type Listing struct {
+type languagesOutput struct {
 	Languages []model.Language `json:"languages"`
 }
 
-// MarshalOutput implements the output.Marshaller interface.
-func (l Listing) MarshalOutput(f output.Format) interface{} {
-	if f == output.PlainFormatName {
-		return l.Languages
-	}
-	return l
+func (o *languagesOutput) MarshalOutput(f output.Format) interface{} {
+	return o.Languages
+}
+
+func (o *languagesOutput) MarshalStructured(f output.Format) interface{} {
+	return o
```

Comment:
This results in `{"languages":[{...}]}`, which is unusual and inconsistent with most of our other output. However, perhaps backwards-compatibility is desirable here, so I left it as-is. I would recommend simply returning a list of languages though.